packages/nc-gui/components/cell/Currency.vue

```diff
@@ -78,7 +78,10 @@ onMounted(() => {
     :ref="focus"
     v-model="vModel"
     type="number"
-    class="w-full h-full text-sm border-none rounded-md outline-none focus:outline-transparent focus:ring-0"
+    class="w-full h-full text-sm border-none rounded-md px-0 py-1 outline-none focus:outline-none focus:ring-0"
+    :class="{
+      '!px-2': isExpandedFormOpen,
+    }"
```

Comment:
most of the time you don't need to / you shouldn't use `!` to override the style. If `px-2` is only for opened expanded form and `px-0` is for the else case. You can simply do in this way. (apply to all)

```ts
:class= "isExpandedFormOpen ? 'px-2' : 'px-0'"
```

ofc remove `px-0` from L81.